libraries/standard/mqtt/integration-test/test_config.h

```diff
@@ -72,16 +72,14 @@
  *
  * In general, port 8883 is for secured MQTT connections.
  */
-#define BROKER_PORT    ( 8883 )
+#define BROKER_PORT                 ( 8883 )
 
 /**
  * @brief Path of the file containing the server's root CA certificate.
  *
  * This certificate should be PEM-encoded.
  */
-#ifndef SERVER_ROOT_CA_CERT_PATH
-    #error "SERVER_ROOT_CA_CERT_PATH should be defined for MQTT system tests."
-#endif
+#define SERVER_ROOT_CA_CERT_PATH    "certificates/mosquitto.org.crt"
```

Comment:
What's wrong with keeping the `#ifndef`? It allows the user to define the macro in cmake without having to change any files

Reply:
Nothing wrong. Not sure why I removed these.